client/src/components/Notification/InWebNotification.tsx

```diff
@@ -0,0 +1,17 @@
+import { useSocket } from '../../context/useSocketContext';
+import { useState } from 'react';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+export default function InWebNotification(): JSX.Element {
+  const { socket } = useSocket();
+  const { updateSnackBarMessage } = useSnackBar();
+  const [notification, setNotification] =
+    useState<{ message: any; title: string; date: Date } | null | undefined>(null);
+
+  socket?.on('notification', (_notification_) => {
```

Comment:
```suggestion
  socket?.on('notification', (notification) => {
```
I think let's stick to this for naming conventions

Reply:
okay. fixed